projects/1.18-fabric/assets/alloy-forgery/alloy_forgery/lang/zh_cn.json

```
@@ -0,0 +1,24 @@
+{
+  "block.alloy_forgery.bricks_forge_controller": "红砖锻造控制器",
+  "block.alloy_forgery.stone_bricks_forge_controller": "石砖锻造控制器",
+  "block.alloy_forgery.deepslate_bricks_forge_controller": "深板岩砖锻造控制器",
+  "block.alloy_forgery.end_stone_bricks_forge_controller": "末地石砖锻造控制器",
+  "block.alloy_forgery.polished_blackstone_forge_controller": "磨制黑石锻造控制器",
+  "block.alloy_forgery.prismarine_bricks_forge_controller": "海晶石砖锻造控制器",
+
+  "tooltip.alloy_forgery.forge_tier": "等级: %d",
+  "tooltip.alloy_forgery.fuel_capacity": "燃料容量: %d",
+
+  "container.alloy_forgery.forge_controller": "合金锻造器",
+
+  "container.alloy_forgery.rei.title": "锻造合金",
+  "container.alloy_forgery.rei.min_tier": "锻造器等级要求: %d",
+  "container.alloy_forgery.rei.fuel_per_tick": "燃料消耗: %d/tick",
+
+  "message.alloy_forgery.invalid_multiblock": "无效的多方块配置",
+  "itemGroup.alloy_forgery.alloy_forgery": "合金锻造",
+  "itemGroup.alloy_forgery.alloy_forgery.button.discord": "Discord页面",
+  "itemGroup.alloy_forgery.alloy_forgery.button.curseforge": "CurseForge页面",
+  "itemGroup.alloy_forgery.alloy_forgery.button.modrinth": "Modrinth页面",
+  "itemGroup.alloy_forgery.alloy_forgery.button.github": "GitHub页面"
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

```suggestion
  "itemGroup.alloy_forgery.alloy_forgery.button.discord": "Discord",
  "itemGroup.alloy_forgery.alloy_forgery.button.curseforge": "CurseForge",
  "itemGroup.alloy_forgery.alloy_forgery.button.modrinth": "Modrinth",
  "itemGroup.alloy_forgery.alloy_forgery.button.github": "GitHub
```